src/env-vars.ts

```diff
@@ -38,9 +38,17 @@ const WECHATY_PUPPET_WECHAT_TOKEN = (value?: string): undefined | string => {
   return process.env['WECHATY_PUPPET_WECHAT_TOKEN']
 }
 
+const WECHATY_PUPPET_WECHAT_UOS = (value?: boolean): boolean => {
+  if (typeof value !== 'undefined') {
+    return value
+  }
+  return String(process.env['WECHATY_PUPPET_WECHAT_UOS']) === '1'
```

Comment:
>   return !!process.env['WECHATY_PUPPET_WECHAT_PUPPETEER_STEALTHLESS']

Like the `STEALTHLESS`, please follow the naming convention and the `!!` boolean check as the following demonstrated:

```diff
const WECHATY_PUPPET_WECHAT_UOS = (value?: boolean): boolean => {
  if (typeof value !== 'undefined') {
    return value
  }
-   return String(process.env['WECHATY_PUPPET_WECHAT_UOS']) === '1'
+   return !!process.env['WECHATY_PUPPET_WECHAT_UOS']
```

Reply:
I change this code to:
```ts
return /^(true|1)$/i.test(String(process.env['WECHATY_PUPPET_WECHAT_PUPPETEER_UOS']))
```
[RegExp.prototype.test](https://developer.mozilla.org/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/RegExp/test) returns boolean type.
It is more flexible for user to enable or disable UOS patch through env variable `WECHATY_PUPPET_WECHAT_PUPPETEER_UOS`, like setting `WECHATY_PUPPET_WECHAT_PUPPETEER_UOS=0` to disable.